Userland/Libraries/LibWeb/Layout/TableFormattingContext.cpp

```diff
@@ -1110,27 +1110,15 @@ void TableFormattingContext::position_cell_boxes()
                 cell_state.padding_top += row_content_height - cell_state.border_box_height();
                 break;
             }
+            case CSS::VerticalAlign::Sub:
+            case CSS::VerticalAlign::Super:
+            case CSS::VerticalAlign::TextBottom:
+            case CSS::VerticalAlign::TextTop:
```

Comment:
I'm not sure these are correct; which spec did you use?

Reply:
Great, please add that spec link to the top of the switch so it's easier to find in the future!

Reply:
Done